README.md

```diff
@@ -1,5 +1,9 @@
 # Coinduction
 
+[![Docker CI][docker-action-shield]][docker-action-link]
+[docker-action-shield]: https://github.com/damien-pous/coinduction/workflows/Docker%20CI/badge.svg?branch=master
+[docker-action-link]: https://github.com/damien-pous/coinduction/actions?query=workflow:"Docker%20CI"
```

Comment:
Sorry, there needs to be a newline between the first and second line here for the badge to work. That is, it should be as follows:
```
[![Docker CI][docker-action-shield]][docker-action-link]

[docker-action-shield]: https://github.com/damien-pous/coinduction/workflows/Docker%20CI/badge.svg?branch=master
[docker-action-link]: https://github.com/damien-pous/coinduction/actions?query=workflow:"Docker%20CI"
```
Maybe easiest if you push this directly.

Reply:
sure, done.